website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,220 @@
+
+### Global filter functions
+
+Depending on the fetching method and complexity of the data set, global filter functions can be used to expose methods that effect the entire data set. Global functions should be differentiated from the primary filters and aligned to the end of the data set. Examples include:
+
+- Manually refreshing the data set
+- Triggering a [Flyout](/components/flyout) with more complex filters
+
+#### Multiple global filter functions
+
+![Multiple global functions](/assets/patterns/filter-patterns/global-filter-functions.png =874x*)
+
+#### Single global function
+
+![Single function](/assets/patterns/filter-patterns/global-filter-functions-single.png =874x*)
+
+In the case a multiple filter functions, use the [Segmented Group](/components/segmented-group). If only a single function is necessary, use the secondary [Button](/components/button#secondary).
+
+### Pagination
+
+Use pagination to break down the filtered data set into pages. For more details, refer to the [Pagination](/components/pagination) guidelines.
+
+![Pagination example](/assets/patterns/filter-patterns/pagination-layout.png =559x*)
+
+## Empty state
+
+Empty state within a filter pattern is expressed in two different ways:
+
+1. Applied filters have resulted in no records being returned in the data set.
+2. No filters have been applied resulting in no tags in the applied filters area.
+
+### Within a data set
+
+Depending on the applied filters, there may not be any records returned from a data set that match parameters and values selected. In this case, communicate the empty state to the user and highlight instructions to adjust the filters or provide a method to clear all filters.
+
+Use the Helios [Application State](/components/application-state) component to communicate the lack of results and steps the user can take to remedy the situation.
+
+![Empty state example](/assets/patterns/filter-patterns/filter-patterns-empty-state.png =559x*)
+
+### Within applied filters
+
+If no filters have been applied, it can be helpful to communicate this in the applied filters area. Use a [Tooltip](/components/tooltip) coupled with the applied filters label to guide the user to the filter functions.
+
+![Empty state in applied filters](/assets/patterns/filter-patterns/applied-filters-empty-state.png =450x*)
+
+## Filtering methods
+
+Depending on the fetching strategy, size, and rendering method of the data set, how filters are applied and the impact on the user experience varies.
+
+### Live filtering
+
+Live filtering refers to updating the records within a data set immediately after a user makes a selection. This method results in a responsive user experience, but may not be possible with large data sets or complex filtering methods.
+
+<video width="100%" controls loop>
+  <source
+    src="/assets/patterns/filter-patterns/live-filtering.mp4"
+    type="video/mp4"
+  />
+</video>
```

Comment:
In the "Empty state" section, you showcase the empty state on applied filters with a tooltip, but that's missing from the video. If that's our recommended approach, then our examples should also showcase that.

Reply:
It looks like all the videos are that way, actually.

Reply:
Updated this!